chia/_tests/util/protocol_messages_json.py

```diff
@@ -1,3 +1,4 @@
+# ruff: noqa: E501
```

Comment:
I moved these ignores from the linter config to the file itself because it seemed better to keep the information with the problematic practices.  The linter `per-file-ignores` to me seems like it should be a pattern of files that are excluded from some rule.